From 4ab0958643827777f526b180ece874d40009986d Mon Sep 17 00:00:00 2001
From: Tomas Korbar <tkorbar@redhat.com>
Date: Thu, 27 Sep 2018 17:27:39 +0200
Subject: [PATCH] Fix TypeError raised by debugprint call

- this error is caused by typo in options.py:424 debugprint call
- debugprint function takes only one parameter so i think it should
have been formated string
- fix https://bugzilla.redhat.com/show_bug.cgi?id=1619593
---
 options.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/options.py b/options.py
index 4dc0b1fe..d756f98d 100644
--- a/options.py
+++ b/options.py
@@ -421,7 +421,7 @@ class OptionSelectOne(Option):
             self.selector.set_active(selected)
         else:
             debugprint("Unknown value for %s: %s" % (name, value))
-            debugprint("Choices:", supported)
+            debugprint("Choices: %s" % (supported))
             if len(supported) > 0:
                 debugprint("Selecting from choices:", supported[0])
                 self.selector.set_active(0)
-- 
2.17.2

